memphis/memphis.py

```diff
@@ -402,7 +432,7 @@ async def produce(self, message, ack_wait_sec=15, headers={}, async_produce=Fals
         """
         try:
             if self.connection.schema_updates_data[self.internal_station_name] != {}:
-                message = self.validate(message)
+                message = self.validateMsg(message)
```

Comment:
just call validate message and make al necessary checks inside, no if statements in here